src/commands/authorize/authorizeCommand.ts

```diff
@@ -0,0 +1,61 @@
+interface AuthorizeInterface {
+    state: string;
+    client_id: string;
+    redirect_uri: string;
+    scope: string;
+    cookie?: string | undefined;
+    sessionToken?: string | undefined;
+}
+
+export default class AuthorizeCommand implements AuthorizeInterface {
+    readonly state: string;
+    readonly client_id: string;
+    readonly redirect_uri: string;
+    readonly scope: string;
+    readonly cookie?: string | undefined;
+    readonly sessionToken?: string | undefined;
```

Comment:
Currently, the session token always start with `20111` (but nothing explain what it means, so I don't know if this can change) and have a length of 55 chars.
We could test the token length.